frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -68,41 +71,58 @@ export class ReduxTlmDispatcher extends React.Component {
     ])
   }
 
+  handleNotification (data) {
+    if (this.props.user.userId !== data.fields.author.user_id) {
+      this.props.dispatch(addNotification(data))
+    }
+  }
+
   handleWorkspaceDeleted = data => {
-    this.props.dispatch(removeWorkspace(data.fields.workspace))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(removeWorkspace(data.fields.workspace))
+    this.handleNotification(data)
   }
 
   handleWorkspaceModified = data => {
-    this.props.dispatch(updateWorkspaceDetail(data.fields.workspace))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(updateWorkspaceDetail(data.fields.workspace))
+    this.handleNotification(data)
   }
 
   handleMemberCreated = data => {
-    this.props.dispatch(addWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(addWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
+    this.handleNotification(data)
   }
 
   handleMemberModified = data => {
-    this.props.dispatch(updateWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(updateWorkspaceMember(data.fields.user, data.fields.workspace.workspace_id, data.fields.member))
+    this.handleNotification(data)
   }
 
   handleMemberDeleted = data => {
-    this.props.dispatch(removeWorkspaceMember(data.fields.user.user_id, data.fields.workspace.workspace_id))
-    if (this.props.user.userId === data.fields.user.user_id) this.props.dispatch(removeWorkspace(data.fields.workspace))
-    this.props.dispatch(addNotification(data))
+    const { props } = this
+    props.dispatch(removeWorkspaceMember(data.fields.user.user_id, data.fields.workspace.workspace_id))
+    if (props.user.userId === data.fields.user.user_id) props.dispatch(removeWorkspace(data.fields.workspace))
```

Comment:
I know you didn't write this line, but isn't the condition strange here? It makes me think that other tabs of a user who removes a workspace won't be updated...

Reply:
@gbonaspetti git says that you wrote this line, can you shed some light please?

Reply:
Re-reading this line, it seems correct. @grignards are you confusing `data.fields.user` with `data.fields.author`?

Here, `data.fields.user` is removed from `data.fields.workspace`. If we are user `data.fields.user`, we should remove the workspace from the UI.

Reply:
Yes, I was confusing author and user. This is OK.